src/Abstractions/RetryPolicy.cs

```diff
@@ -123,11 +133,56 @@ public class RetryPolicy
     /// </value>
     public TimeSpan RetryTimeout { get; }
 
+    /// <summary>
+    /// Gets a delegate to call on exception to determine if retries should proceed.
+    /// </summary>
+    /// <value>
+    /// Defaults delegate that always returns true (i.e., all exceptions are retried).
+    /// </value>
+    public Func<Exception, bool> Handle { get; private set; }
```

Comment:
Can the `private set` be removed?
```suggestion
    public Func<Exception, bool> Handle { get; }
```

Reply:
I could not get the code to compile when removing the `private set`. Compiler said the property cannot be set since it was read only.  Keeping with the immutability of this class, I changed this to `private init`.